### DISCUSSION TITLE
"Thinner Ubuntu fonts" in Mint 22 release notes

### DISCUSSION THREAD

**Discussion by billyswong** [linuxmint/discussions#546] [Ideas]
In current Mint 22 release notes, it wrote

> Going back to the old fonts is possible by downloading the old fonts package and refusing updates for it. For more information read https://askubuntu.com/questions/1465216/install-the-old-version-of-the-ubuntu-font. 

But that askubuntu webpage is providing less-than-ideal outdated information. In current Ubuntu 24.04 release notes, it wrote

> Updated Ubuntu font
> 
> A more modern slimmer version of the Ubuntu font family is now shipped as standard. Anyone wishing to return to the older Ubuntu font used in 22.04 can do so by installing the `fonts-ubuntu-classic` package.

which is cleaner and easier to apply.

**Comment by user414** [comment]
@billyswong Thanks, it's hard to believe how much time I wasted investigating this. I considered a lot of potential root cause but never thought of the Ubuntu team just changing the font without renaming it. As others suggested it's difficult to imagine the rationale for simply changing the fonts instead of making a new font like "Ubuntu slim" and making that the default in Ubuntu. 

To make matter worse I use XFCE and the font weight are incorrectly applied, and it seem to be an issue that has been lingering for a while and is unlikely to be resolved soon, in a normal world one could have simply moved to medium or bold weight but that does not work.

**Comment by clefebvre** [comment]
fonts-ubuntu-classic is a bad solution, that's why it's not mentioned in the Mint release notes.

**Reply by billyswong** [reply]
Hi, anywhere may I read on the badness of fonts-ubuntu-classic, in comparison to the topmost suggestion in askubuntu?

**Reply by ra50** [reply]
Also curious why it's a bad solution.

**Reply by bastschonet** [reply]
Clement, which solution is not bad?
I don't like the thinner fonts...

For my impaired sight the old ones are better readable.

**Reply by nicolashainaux** [reply]
I'd like to know, too, why fonts-ubuntu-classic is a bad solution.
